vmdb/lib/extensions/ar_virtual.rb

```diff
@@ -352,26 +352,34 @@ def preloaders_on_with_virtual(association, records, preload_scope = nil)
   end
 
   module FinderMethods
-    def find_with_associations_with_virtual
-      original, self.includes_values = includes_values, klass.remove_virtual_fields(includes_values) if includes_values
+    def without_virtual_includes
+      if includes_values
+        spawn.without_virtual_includes!
```

Comment:
It's a special `clone`. This pair of methods is a [standard pattern](https://github.com/rails/rails/blob/master/activerecord/lib/active_record/relation/query_methods.rb#L139-L150) in Relation.